README.md

```diff
@@ -153,8 +153,27 @@ downloader.onProgress = (bytesDownloaded: number, bytesTotal: number) => {
 
 ## Error List
 
-| Code         | Message                      | Reason                                                                      |
-| ------------ | ---------------------------- | --------------------------------------------------------------------------- |
-| UNAUTHORIZED | You can't download this file | Trying to download a file which is neither owned by you nor shared with you |
-| TRANSACTION  | \*                           | Smart contract Errors                                                       |
-| TRANSACTION  | No space left for user       | You have already consumed your storage or bandwidht limit                   |
+| Code         | Message                                                  | Reason                                                                      |
+| ------------ | -------------------------------------------------------- | --------------------------------------------------------------------------- |
+| UNAUTHORIZED | You can't download this file                             | Trying to download a file which is neither owned by you nor shared with you 
+| TRANSACTION  | This function can only be called by file owner           | Only owner of the file have access to the function i.e, either to delete, revoke or transfer file 
```

Comment:
@naraharirohith Grammar - "Only the owner of the file has access to the function.. "